spec/stat_tracker_spec.rb

```diff
@@ -38,17 +36,47 @@
 
     it "finds highest total score" do
       expect(@stat_tracker.highest_total_score).to eq(7)
-   end
+    end
 
     it 'can return the lowest score' do
       expect(@stat_tracker.lowest_total_score).to eq(1)
     end
 
-    it "returns the percentage of tied games" do
-      expect(@stat_tracker.percentage_ties).to eq(17.8)
+    it "finds highest total score" do
+      expect(@stat_tracker.highest_total_score).to eq 7
+    end
+        
+    it 'can return the lowest score' do 
+      expect(@stat_tracker.lowest_total_score).to eq(1)
     end
 
+    it "tracks wins" do
+      expect(@stat_tracker.game_wins).to eq 29
+    end
 
-  end
+    it "tracks losses" do
+      expect(@stat_tracker.game_losses).to eq 29
+    end
+
+    it "tracks home games" do
+      expect(@stat_tracker.home_games).to eq 30
+    end
+
+    it "tracks away games" do
+      expect(@stat_tracker.away_games).to eq 30
+    end
+
+    it "calculates home wins" do
+      expect(@stat_tracker.home_wins).to eq 21
+    end
+
+    it 'calculates percentage wins' do 
+      expect(@stat_tracker.percentage_home_wins).to eq 70
+    end
+
+    it "returns the percentage of tied games" do
+      expect(@stat_tracker.percentage_ties).to eq(17.8)
+    end
+  end 
```

Comment:
Thanks for writing a test for all the helper methods too!! Should we have the `calculates percentage wins` eq value show it is a float by having it be 70.0?

Reply:
great idea! I changed the spec.